tests/TestCases/UnitTestCase.php

```diff
@@ -12,4 +12,18 @@
  */
 abstract class UnitTestCase extends BaseUnitTestCase
 {
+    /**
+     * Returns private property value.
+     *
+     * @param mixed $object
+     * @param string $property
+     *
+     * @return mixed
+     */
+    protected function getPrivatePropertyValue($object, string $property)
```

Comment:
I am against adding this general function to the test case. Relying on private access is not a suitable solution.